packages/styles/stories/BTP/Navigation/horizontal/navigation-horizontal-expander.example.html

```diff
@@ -1,19 +1,18 @@
 <div style="height: 500px">
     <nav class="fd-navigation fd-navigation--horizontal" aria-label="Horizontal Navigation">
         <div class="fd-navigation__container">
-            <ul class="fd-navigation__list" role="menubar" aria-label="Horizontal Navigation">
+            <ul class="fd-navigation__list" role="tablist" aria-orientation="horizontal" aria-label="Horizontal Navigation">
```

Comment:
the default aria-orientation for role="tablist" is horizontal, you don't need to specify it (just FYI)

Reply:
UI5 does